static/css/calendar-icon.less

```diff
@@ -0,0 +1,92 @@
+/* ==========================================================================
+   cfgov-refresh
+   calendar-icon
+   ========================================================================== */
+
+/* topdoc
+  name: Calendar Icon
+  family: cfgov-calendar-icon
+  notes:
+    - "A square-ish calendar icon with a three-letter month name and date."
+  patterns:
+    - name: Calendar Icon
+      markup: |
+        <time class="calendar-icon" data-month="Jan" data-day="21" datetime="2015-01-21">
+          <span class="u-visually-hidden">Jan 21, 2015</span>
+        </time>
+        denotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          time.calendar-icon
+            span.u-visually-hidden
+  tags:
+    - cfgov-calendar-icon
+*/
+
+
+/*  Mixin to generate a calendar icon.
+    @month [String] The month (3-letter abbreviation, typically).
+    @date [Number] The day in the month.
+    @background-color [Color] A color value for the icon background (behind the month).
+    @foreground-color [Color] A color value for the color behind the day.
+    @scale [Number] A multiplier for the size of the icon (1 is the default). */
+.cf-calendar-icon( @month, @date, @background-color, @foreground-color, @scale ) {
+    /* Display & Box Model */
+    display: inline-block;
+    min-width: 3em * @scale;
+    border: 0.25em solid @background-color;
+    border-radius: 0.375em * @scale;
+
+    /* Background */
+    background-color: @background-color;
+
+    /* Foreground */
+    font-family: 'Avenir Next', Helvetica, Arial;
```

Comment:
This should be replaced with the `.webfont-regular()` mixin or it will look for a locally installed font.